Actually, I just found an error in the log:

```
  Traceback (most recent call last):
  File "/home/vagrant/myconnectome/myconnectome/scripts/run_everything.py", line 67, in <module>
   execfile('%s/scripts/rnaseq_analysis.py'%basepath)
   File "/home/vagrant/myconnectome/myconnectome/scripts/rnaseq_analysis.py", line 71, in <module>
    regress_rin_pca.regress_rin_pca()
    File "build/bdist.linux-x86_64/egg/myconnectome/rnaseq/regress_rin_pca.py", line 15, in regress_rin_pca
  IOError: [Errno 2] No such file or directory: '/home/vagrant/myconnectome/rna-seq/varstab_data_prefiltered.txt'
```

and the last output in the log is:

```
  ds031/RNA-seq/rin.txt
  /home/vagrant/myconnectome/rna-seq/rin.txt
  ds031/RNA-seq/drawdates.txt
  /home/vagrant/myconnectome/rna-seq/drawdates.txt
  ds031/RNA-seq/pathsubs.txt
  /home/vagrant/myconnectome/rna-seq/rnaseq-subcodes.txt
```

I don't have any insight about this at the moment.

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -94,6 +94,7 @@ def start_analyses():
         my_env = os.environ.copy()
         my_env['MYCONNECTOME_DIR'] = '/home/vagrant/myconnectome'
         my_env['WORKBENCH_BIN_DIR'] = '/usr/bin'
+        os.chdir('/home/vagrant/myconnectome')
         p = Popen(['/home/vagrant/miniconda/bin/python', '/home/vagrant/myconnectome/myconnectome/scripts/run_everything.py'],
                    stdout=open('/home/vagrant/myconnectome/myconnectome_job.log', 'w'),
                    stderr=open('/home/vagrant/myconnectome/myconnectome_job.err', 'a'),

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -11,6 +11,7 @@ then
  chmod +x miniconda.sh
  ./miniconda.sh -b
  echo "export PATH=$HOME/miniconda/bin:\\$PATH" >> .bashrc
+ echo "export PATH=$HOME/miniconda/bin:\\$PATH" >> .env
 fi
 
 # install nipype dependencies
@@ -49,6 +50,7 @@ mkdir $HOME/R_libs
 sudo mkdir /var/www
 sudo mkdir /var/www/results
 echo "export R_LIBS_USER=$HOME/R_libs" >> .bashrc
+echo "export R_LIBS_USER=$HOME/R_libs" >> .env
 fi
 
 
@@ -57,6 +59,8 @@ then
   git clone https://github.com/poldrack/myconnectome.git $HOME/myconnectome
   echo "export MYCONNECTOME_DIR=$HOME/myconnectome" >> .bashrc
   echo "export WORKBENCH_BIN_DIR=/usr/bin" >> .bashrc
+  echo "export MYCONNECTOME_DIR=$HOME/myconnectome" >> .env
+  echo "export WORKBENCH_BIN_DIR=/usr/bin" >> .env
   cd $HOME/myconnectome
 fi
 
@@ -267,9 +271,11 @@ user = vagrant
   sudo cp /tmp/abcde /etc/supervisor/conf.d/flask_project.conf
 fi
 
+# Install my connectome and start analyses
 cd /home/vagrant/myconnectome
 $HOME/miniconda/bin/python /home/vagrant/myconnectome/setup.py install
 
+# Start the flask application via supervisor
 sudo ln -s /etc/nginx/sites-available/flask_project /etc/nginx/sites-enabled/flask_project
 cd /var/www
 sudo /etc/init.d/nginx restart
@@ -277,6 +283,12 @@ sudo supervisorctl reread
 sudo supervisorctl update
 sudo supervisorctl start flask_project
 echo "Open browser to 192.168.0.20:5000"
+
+# Start the analysis for the user
+touch /home/vagrant/myconnectome/.started
+source /home/vagrant/.env
+$HOME/miniconda/bin/python /home/vagrant/myconnectome/myconnectome/scripts/run_everything.py > /home/vagrant/myconnectome/myconnectome_job.out 2> /home/vagrant/myconnectome/myconnectome_job.err &
+
 
 SCRIPT
 

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -94,6 +94,7 @@ def start_analyses():
         my_env = os.environ.copy()
         my_env['MYCONNECTOME_DIR'] = '/home/vagrant/myconnectome'
         my_env['WORKBENCH_BIN_DIR'] = '/usr/bin'
+        my_env['R_LIBS_USER'] = '/home/vagrant/R_libs'
         os.chdir('/home/vagrant/myconnectome')
         p = Popen(['/home/vagrant/miniconda/bin/python', '/home/vagrant/myconnectome/myconnectome/scripts/run_everything.py'],
                    stdout=open('/home/vagrant/myconnectome/myconnectome_job.log', 'w'),

